nextstrain/cli/runner/aws_batch/__init__.py

```diff
@@ -54,6 +61,20 @@ def register_arguments(parser) -> None:
         metavar = "<name>",
         default = DEFAULT_S3_BUCKET)
 
+    development.add_argument(
+        "--aws-batch-cpus",
+        dest    = "cpus",
+        help    = "Number of vCPUs to request for job",
+        metavar = "<name>",
+        default = DEFAULT_CPUS)
+
+    development.add_argument(
+        "--aws-batch-memory",
+        dest    = "memory",
+        help    = "Amount of memory in MB to request for job",
+        metavar = "<name>",
```

Comment:
`metavar` should be `<megabytes>` or `<number>` instead of `<name>`.

Reply:
I see. I hadn't ever used `metavar` before. Just read the docs.